.coderabbit.yaml

```
@@ -0,0 +1,75 @@
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+  tools:
+    checkov:
+      enabled: true
+    eslint:
+      enabled: true
+    gitleaks:
+      enabled: true
+
+    checkov:
```

yamllint

[error] duplication of key "checkov" in mapping

(key-duplicates)